package.json

```diff
@@ -39,9 +41,18 @@
   },
   "author": "Adrian Sinclair <adrian@transloadit.com>",
   "license": "MIT",
+  "bin": "./bin/cmd.js",
```

Comment:
If you refer to this file as `lib/cli.js` you can put it in `src` and transpile it the same way as your other sources